scripts/language_model/index.rst

```diff
@@ -232,9 +232,9 @@ Results using `xlnet_12_768_12`:
 +-----------------+---------------------+-----------------------+--------------------------------------------------------------------------------------------------------------------------------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------+
 | MRPC            |Accuracy/F1          |87/90                  |`log <https://github.com/dmlc/web-data/tree/master/gluonnlp/logs/language_model/xlnet_l12_h768_a12_finetuned_MRPC.log>`__                   |`command <https://github.com/dmlc/web-data/tree/master/gluonnlp/logs/language_model/xlnet_l12_h768_a12_finetuned_MRPC.sh>`__                                     |
 +-----------------+---------------------+-----------------------+--------------------------------------------------------------------------------------------------------------------------------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------+
-| STS-B           |Pearson Corr.        |86                     |`log <https://github.com/dmlc/web-data/tree/master/gluonnlp/logs/language_model/xlnet_l12_h768_a12_finetuned_STS-B.log>`__                  |`command <https://github.com/dmlc/web-data/tree/master/gluonnlp/logs/language_model/xlnet_l12_h768_a12_finetuned_STS-B.sh>`__                                    |
```

Comment:
Needs update based on https://github.com/dmlc/web-data/pull/223 https://github.com/dmlc/web-data/pull/224 ?

Reply:
I think it is already updated to 89.34 in the following lines?

Reply:
Yes. I wrongly assumed the link needs update as well. But the old files were overwritten (which is fine, as we didn't do a release that would link the old files).